### PR TITLE
Moving media presenter out of app

### PR DESCRIPTION
**Pull request by liviaw**
Hi this Pr consist of changes on:

- create an instance of media presenter inside index.tsx and pass into App.css
- no visual changes

known:
I have not change the variable show into showModal in this PR

### CODE REVIEW

**Comment by juliamitchelmore** [comment]
Please request review from https://github.com/SamuelDrinkwater 👍

File: src/Components/MediaPresenter.ts [hunk 1]
```diff
@@ -195,3 +195,5 @@ export class MediaPresenter {
     );
   }
 }
+
+const mediaPresenter = new MediaPresenter();
```
Comment (SamuelDrinkwater) [inline src/Components/MediaPresenter.ts]: This isn't exported or used in this file. You could either remove this line or export it and import the instance in `App.tsx`

**Review by SamuelDrinkwater — CHANGES_REQUESTED** [review]
Mostly looks good. Fix up the presenter import/export and then 👍

**Review by SamuelDrinkwater — APPROVED** [review]
Nice work 🎉

**Pull request merged**